package.json

```
@@ -83,13 +86,16 @@
     "standard-version": "^7.1.0",
     "svelte": "^3.22.2",
     "svelte-spa-router": "^2.1.0",
+    "ts-jest": "^26.0.0",
     "typescript": "3.8.3",
     "wait-on": "^5.0.0"
   },
   "scripts": {
     "start": "run-p --race svelte:watch proxy:start electron:start",
     "test": "TZ='UTC' run-p --race proxy:start:test wait:test",
     "test:debug": "TZ='UTC' run-p --race svelte:watch proxy:start:test wait:debug",
+    "jest": "jest",
```

```suggestion
    "test:unit": "jest",
```

I'd recommend that we move to intention over technology names, this way we have stable commands no matter what runners are used under the hood.

---

Agree completely - I'm using `jest` so the cypress tests aren't affected by this experiment but I think `test:unit` should be the command we end up using